tests/test_validate_edf.py

```diff
@@ -15,13 +15,10 @@
 def test_validate_edf():
     # force a no file error with Path = None
     # assert make_ballots.make_ballots(_edf=None) == NO_FILE
-    assert validate_edf.validate_edf(_edf=None) == NO_FILE
+    assert validate_edf(_edf=None) == NO_FILE
     # ensure not_a_file is actually not a file
-    assert Path.is_file(not_a_file) == False
-    assert validate_edf.validate_edf(not_a_file) == NO_FILE
-    # test empty json file
-    # assert Path.is_file(empty_file_path)
-    # assert validate_edf.validate_edf(empty_file_path) == NO_DATA
+    assert not_a_file.is_file() == False
```

Comment:
```suggestion
    assert not not_a_file.is_file()
```

Reply:
The double negative was too much for my brain, so I changed it like this:

```
assert not imaginary_file.is_file()
```